tests/testhelpers/clusterHelpers.go

```diff
@@ -22,10 +22,10 @@ func CreateLocalTestCluster() (*setup.Cmds, error) {
 	builder := setup.NewGoBuilder(clusterCmds)
 	go func() {
 		sched := map[string]setup.SchedulerStrategy{
-			"local.local": setup.NewLocalLocal(nil, builder, clusterCmds),
+			"local.local": setup.NewLocalLocal(nil, builder, clusterCmds, log.DebugLevel),
```

Comment:
Is DebugLevel the right choice here? I don't actually know, but we for sure don't want debug in TravisCI.